### ISSUE TITLE
ADB connection reset error when running gradle RunFragmention on a windows machine

### ISSUE THREAD

**Issue by arvind1017** [testvagrant/optimusTemplate#20]
While setting up Optimus on a windows 7/10 machine, I am getting a connection reset exception, when the framework is trying to run the shell command "Wm size" through ADB.

[Optimus error log.txt](https://github.com/testvagrant/optimusTemplate/files/1801072/Optimus.error.log.txt)

Attached is the error log i get.

**Comment by sarathsp23** [comment]
I am seeing the same error , for my windows machine as well. It works fine on my mackbook. Also optimus setup or choco did not install the mongoDB correctly. I had to install it separately.

**Comment by nishantverma** [comment]
We have added this as a bug and look into it. Thanks for giving it a try!

**Comment by arvind1017** [comment]
team, did you get a chance to look into this issue?

**Comment by KrishnB** [comment]
Can you try downgrading appium version once? To 1.7.0 and see if this happens?

**Comment by arvind1017** [comment]
Hi Krishna,

Thanks for the response. I tried on appium versions 1.6.3, 1.7.0 and 1.7.2, but no success.

Thanks
Arvind

**Comment by KrishnB** [comment]
Thanks for the update Arvind. Will look into this and update you. Can I assume that this doesn't happen on Distribution mode?

**Comment by arvind1017** [comment]
Since I was not able to set it up on one mobile, didnt give a try to have the distribution mode. I would think that, its not a safe assumption to make

**Comment by KrishnB** [comment]
@arvind1017 @sarathsp23 We had reproduced this on a windows machine and were able to resolve it by downgrading appium version to 1.7.0 however there was an issue with appium log path on windows which is fixed in optimus version 3.0 below are the dependencies.

### Optimus Dependency
```
repositories {
    mavenCentral()
    jcenter()
}
dependencies {
    compile group: 'com.testvagrant', name: 'optimus', version: '3.0'
}
```

### Optimus Gradle plugin dependency
```
buildscript {
    repositories {
        maven {
            url "https://plugins.gradle.org/m2/"
        }
        mavenCentral()
    }
    dependencies {
        classpath "gradle.plugin.com.testvagrant.optimus:OptimusGradle:3.0"
    }

}
```

Let us know if this helps.
